cypress/e2e/requests/edit-request.cy.js

```diff
@@ -38,15 +38,15 @@ describe('Requests', () => {
     'C556 Request: Edit requests. Make sure that edits are being saved. (vega)',
     { tags: ['smoke', 'vega', 'system', 'shiftLeft'] },
     () => {
-      cy.loginAsAdmin({ path: TopMenu.requestsPath, waiter: Requests.waitLoading });
       [
         EditRequest.requestStatuses.AWAITING_DELIVERY,
         EditRequest.requestStatuses.AWAITING_PICKUP,
         EditRequest.requestStatuses.IN_TRANSIT,
         EditRequest.requestStatuses.NOT_YET_FILLED,
       ].forEach((status) => {
+        cy.loginAsAdmin({ path: TopMenu.requestsPath, waiter: Requests.waitLoading });
         EditRequest.checkIsEditsBeingSaved(requestData, instanceData, status);
-        EditRequest.resetFiltersAndReloadPage();
+        // EditRequest.resetFiltersAndReloadPage();
```

Comment:
do you need this commented line?

Reply:
yes, this method will be refactored soon, I need to keep it here